test/test_automl/test_estimators.py

```diff
@@ -685,6 +713,7 @@ def test_cv_regression(tmp_dir, dask_client):
     score = r2(Y_test, predictions)
     assert score >= 0.1, print_debug_information(automl)
     assert count_succeses(automl.cv_results_) > 0, print_debug_information(automl)
+    assert includes_train_scores(automl.performance_over_time_.columns) is True
 
```

Comment:
Why does this one not have `performance_over_time_is_plausible`? Is it to do with having `cv_models`?